packages/main/src/TableRow.ts

```diff
@@ -137,13 +137,17 @@ class TableRow extends TableRowBase {
 
 		if (eventOrigin === this && this._isInteractive && isEnter(e)) {
 			this.toggleAttribute("_active", true);
-			this._table?._onRowClick(this);
+			this._onclick();
 		}
 	}
 
 	_onclick() {
-		if (this._isInteractive && this === getActiveElement()) {
-			this._table?._onRowClick(this);
+		if (this === getActiveElement()) {
+			if (this._isSelectable && !this._hasRowSelector) {
```

Comment:
This would mean that you cannot have a row-click event in combination with the new selection behavior?
Is this desired? Not sure how the UI5 tables solve this

Reply:
If it is by design, it should be documented imo

Reply:
Yes this is what we discussed with Jens and Oliver. it is similar with the ListBase and it is documented in the [row-click](https://github.com/SAP/ui5-webcomponents/pull/11520/files#diff-c1eacd43766827fdb6c05695d82eb71acd650e1285bfc0287fb7bba82a89cc3cR194-R196) and [enum](https://github.com/SAP/ui5-webcomponents/pull/11520/files#diff-eaed4e5b8370cf5b58faa6e950406680bd717c26dd9b51bfd19185ac28338e0dR17) 

Other things we discussed:
For multi selection there is no interactive header row for (de)select all.
Space/Enter on the cell does not select the row although it selects in grid/Table